pyproject.toml

```diff
@@ -31,6 +31,7 @@ dependencies = [
     "openai>=1.66",
     "prompt-toolkit",
     "python-dotenv",
+    "python-jose",
```

Comment:
not happy about this right now, I am going to split off Auth (and then Credentials) to proper APIs which have proper providers so the overall distribution dependency system takes over. we dont want starter distros to have complex dependencies if they don't need them. 

but for now, this will do.